rewrite-pattern.js

```diff
@@ -810,15 +810,11 @@ const validateOptions = (options) => {
 			case 'unicodePropertyEscapes':
 			case 'unicodeSetsFlag':
 			case 'namedGroups':
+			case 'modifiers':
 				if (value != null && value !== false && value !== 'transform') {
 					throw new Error(`.${key} must be false (default) or 'transform'.`);
 				}
 				break;
-			case 'modifiers':
```

Comment:
This is a breaking change -- can we just silently ignore 'parse'?

Reply:
OK. I left a todo item. If we are going to do a major bump because of other breaking changes, this option can also be removed at that time.